packages/rest-typings/src/v1/users.ts

```diff
@@ -367,6 +367,12 @@ export type UsersEndpoints = {
 	'/v1/users.deleteOwnAccount': {
 		POST: (params: { password: string; confirmRelinquish?: boolean }) => void;
 	};
+
+	'/v1/users.isMuted': {
+		GET: (params: { userId?: string; username?: string; roomId: string }) => {
```

Comment:
Please update this too after doing what's needed to use ajv
(Check how `UsersInfoParamsGet` is used in this file for reference)